templates/registration/login.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Login</h1>
+  {% if form.errors %}
+    <p style="color: red">Invalid credentials</p>
+  {% endif %}
+  <form action="{% url 'login' %}" method="post">
+  {% csrf_token %}
+    {{ form.as_p }}
+    <br>
+  <input type="submit" value="Submit">
+  <input type="hidden" name="next" value="{{ next }}">
+  </form>
+{% endblock %}

```

Comment:
Make sure you've added a blank line at the end of all your files, including '.html' files, as per the checklist requirement.